website/content/en/configurations/ingress-tls.md

```diff
@@ -1,6 +1,12 @@
-# Using Secrets Store CSI and Azure Key Vault Provider to Enable NGINX Ingress Controller with TLS
+---
+type: docs
+title: "Using Secrets Store CSI and Azure Key Vault Provider to Enable NGINX Ingress Controller with TLS"
```

Comment:
Should we shorten this title? It's really long compared to other titles

```suggestion
title: "Enable NGINX Ingress Controller with TLS"
```

Reply:
yeah, sounds good!